systemtest/helpers.bash

```diff
@@ -291,9 +291,21 @@ start_registry() {
         reg_args+=( -e REGISTRY_STORAGE_DELETE_ENABLED=true)
     fi
 
+    # FIXME! TEMPORARY! (As of 2020-03-30)
```

Comment:
Could you add TODO here please?

Reply:
Done, force-pushed, and tests green. Thanks for the feedback.